### ISSUE TITLE
Not running

### ISSUE THREAD

**Issue by OlivierCU** [mgth/LittleBigMouse#307]
application does not run (probably skill issue), doesn't work after restarting pc either

**Comment by JTM712** [comment]
Same here. The application just doesn't launch. It'll show up on the background processes part of task manager but won't go past that

**Comment by fortypercenttitanium** [comment]
Having the same problem here

**Comment by NasvekL** [comment]
Same

**Comment by just4747** [comment]
Same, hasnt worked since the first recent updates. Op hasn't replied to my issue thread about it either for weeks. App is useless now.

**Comment by steakfrosch322** [comment]
Same here, it's just showing up in task manager and not opening a window like it should

**Comment by fortypercenttitanium** [comment]
It seems like the developer is not going to respond to the issue. I found a suitable free alternative for my needs: https://cursr.app/

It doesn't have as many features out of the box but at least the mouse movement between my monitors isn't jacked up.

**Comment by JTM712** [comment]
> It seems like the developer may not respond to the issue, or at very least we shouldn't expect a fix imminently. I found a suitable free alternative for my needs: https://cursr.app/
> 
> It doesn't have as many features out of the box but at least the mouse movement between my monitors isn't jacked up.

Does it fix the scaling between monitors? I tried t once but it didn’t allow me to custom scale my monitors and that screwed up my mouse movements still. It only allowed me to change the position of my monitors

**Comment by guri172** [comment]
4.2.7124.42685 version work, because of lastest version doesn't work.

**Comment by Gianeh** [comment]
> 4.2.7124.42685 version work. lastest version doesn't work.

I doesn't on my side on Windows 10!

**Comment by bevellon** [comment]
The problem is that the program requires admin credentials or UAC prompt to run. If enabled for startup it will not run for a regular user. I have an admin account separate from my main account. The default task scheduler config is to run on admin login. I tried to change this to run at startup for all users, but had the same issue where it will run in the background without doing anything. The program runs as expected with admin login, no intervention needed. I also tried to change the manifest to run as invoker on my main account, but again the program would run and not do anything.
